jest.config.json

```diff
@@ -0,0 +1,26 @@
+{
+  "rootDir": "./",
+  "testMatch": [
+    "**/src/**/?(*.)(spec|test).(ts)?(x)"
+  ],
+  "testPathIgnorePatterns": [
+    "<rootDir>/src($|/.*)(.stories.spec.tsx)$"
+  ],
+  "modulePaths": ["<rootDir>/node_modules"],
+  "coveragePathIgnorePatterns": ["<rootDir>/node_modules/","<rootDir>/es/","<rootDir>/cjs/"],
+  "moduleFileExtensions": ["js", "jsx", "ts", "tsx", "json"],
+  "moduleNameMapper": {
+    "\\.(css|scss)$": "identity-obj-proxy"
+  },
+  "globals": {
+    "__DEBUG__": false,
+    "__DEV__": false,
+    "environment": ""
```

Comment:
Are these being set or is this because it's copied from somewhere?

Reply:
coppied from `core` it was still WIP but thanks